var/spack/repos/builtin/packages/py-gluon-cv/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyGluonCv(PythonPackage):
```

Comment:
I would use the name `py-gluoncv` instead of `py-gluon-cv`. That's how the package is named on PyPI. Personally, I prefer downloading all Python packages from PyPI (when possible) and using consistent naming with PyPI.

Reply:
Closing PR and will resubmit with new name. (It's just easier with my workflow)